estate/models/estate_property.py

```diff
@@ -0,0 +1,85 @@
+from odoo import models, fields, api
+from datetime import date
+from dateutil.relativedelta import relativedelta
+from odoo.exceptions import UserError
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Property"
+
+    name = fields.Char(string="Title", required=True)
+    Property_Type = fields.Text()
+    description = fields.Text()
+    postcode = fields.Char()
+    state = fields.Text()
+    date_availability = fields.Date(copy=False, readonly=True, default=lambda self: date.today() + relativedelta(months=3))
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(readonly=True, copy=False)
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer(string='Living Area (m2)')
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer(string='Garden Area (m2)')
+    garden_orientation = fields.Selection(
+        string='Garden Orientation',
+        selection=[('north', 'North'), ('south', 'South'),
+        ('east', 'East'), ('west', 'West')])
+    active = fields.Boolean(default=True)
+    status = fields.Selection(
+    copy=False,
+    readonly=True,
+    default='new',
+    string='Status',
+    selection=[
+        ('new', 'New'),
+        ('offer_received', 'Offer Received'),
+        ('offer_accepted', 'Offer Accepted'),
+        ('sold', 'Sold'),
+        ('cancelled', 'Cancelled')]
+    )
+
+    def action_set_sold(self):
+        if (self.state != "Cancelled"):
```

Comment:
No need of the parenthesis. The != evaluate if the two elements are similar and will return a boolean value in any case. 
```suggestion
        if self.state != "Cancelled":
```